### PR TITLE
fix: update main component and refactor

### PR DESCRIPTION
**Pull request by spaenleh**
Update main component and add a pagination to the home page.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -50,9 +52,11 @@ export const App = (): JSX.Element => {
 
   return (
     <Routes>
-      <Route path={buildMainPath()} element={<ItemPage />} />
-      <Route path={HOME_PATH} element={<HomePageWithAuthorization />} />
-      <Route element={<Navigate to={HOME_PATH} />} />
+      <Route element={<PageWrapper />}>
+        <Route path={buildMainPath()} element={<ItemPage />} />
+        <Route path={HOME_PATH} element={<HomePageWithAuthorization />} />
+        <Route path="*" element={<Navigate to={HOME_PATH} />} />
```
Comment (ReidyT) [inline src/App.tsx:58]: If no routes path are matched, it redirect the user to the home page ? Shouldn't be clearer to let the user on the current page and show him a 404 error ?

File: src/config/selectors.ts [hunk 2]
```diff
@@ -48,6 +48,10 @@ export const OWN_ITEMS_GRID_ID = 'ownItemsGrid';
 export const buildMemberAvatarClass = (id?: string): string =>
   `memberAvatar-${id}`;
 
+export const HOME_PAGE_PAGINATION_ID = 'honmePagePagination';
```
Comment (ReidyT) [inline src/config/selectors.ts:51]:
```suggestion
export const HOME_PAGE_PAGINATION_ID = 'homePagePagination';
```

File: src/modules/pages/HomePage.tsx [hunk 3]
```diff
@@ -1,128 +1,48 @@
...
 
 const { useAccessibleItems } = hooks;
 
 const PAGE_SIZE = 20;
 
 const HomePage = (): JSX.Element => {
   const { t } = usePlayerTranslation();
-  const navigate = useNavigate();
 
   const [page, setPage] = useState(1);
```
Comment (ReidyT) [inline src/modules/pages/HomePage.tsx:22]: Is the page state still needed in the `Pagination` component, or it is possible to only pass the page as a props and to update the state in the parent using the callback ?

**Review by ReidyT — APPROVED** [review]
LGTM 👍 I just let some comments 😄

File: src/modules/item/ItemForbiddenScreen.tsx [hunk 4]
```diff
@@ -25,40 +22,33 @@ const ItemForbiddenScreen: FC = () => {
...
+        {
+          // todo: add translations
+          t('You cannot access this item')
+        }
+      </Typography>
+      <Typography variant="body1">
+        {
+          // todo: add translations
+          t(
+            'Your current account does not have the rights to access this item.',
```
Comment (pyphilia) [inline src/modules/item/ItemForbiddenScreen.tsx:46]: Use a constant/key? not sure if we do it currently.

**Review by pyphilia — APPROVED** [review]
LGTM 🚥

File: src/modules/pages/HomePage.tsx (hunk 3 above)
Comment (spaenleh) [reply src/modules/pages/HomePage.tsx:22]: I use a controlled version of the pagination component. So the state is needed here. It serves also to change the query key. I am not sure what you were suggesting to do however.

**Pull request merged**